index.md

```diff
@@ -1,34 +1,25 @@
+Anbox Cloud enables running Android apps on any cloud platform at scale. It uses [LXD containers](https://linuxcontainers.org/) to run the nested Android containers and [Juju](https://charmhub.io/) for deployment in a cloud environment.
 
-Using container technology, Anbox Cloud manages the application life cycle while optimising for high density, performance and fast container boot times. It is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures. Deploying the solution is simple and straightforward, and it is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
+Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures. 
 
-Anbox Cloud uses system containers to emulate Android systems. This method results in an isolation and security level similar to virtual machines with an overhead as low as process containers. Therefore, Anbox Cloud can provide at least twice the container density compared to other Android emulation solutions, and can serve up to 100 Android instances per server.
+Since Anbox Cloud uses containerisation to emulate Android systems, you can achieve the isolation and security level of a virtual machine without the associated overhead. Therefore, compared to other Android emulation solutions, Anbox Cloud can provide at least twice the container density and can serve up to 100 Android instances per server.
 
-You should consider using Anbox Cloud if you want to deliver mobile applications that don't depend on the capabilities of your user's devices and that offload compute, storage and energy-intensive applications from the end device to the cloud. Typical use cases include mobile game streaming services, corporate application streaming, application automation and Android application testing.
+Due to its highly scalable nature and performance optimisation, delivering device-agnostic mobile applications becomes very easy. Popular use cases of Anbox Cloud include mobile game streaming services, corporate application streaming, application automation and testing.
 
+# In this documentaion
 | | |
 |--|--|
 |  [Tutorials](https://discourse.ubuntu.com/t/tutorials/28826)</br>  Get started - a hands-on introduction to Anbox Cloud for new users </br> |  [How-to guides](https://discourse.ubuntu.com/t/how-to-guides/28827) </br> Step-by-step guides covering key operations and common tasks |
 |  [Explanation](https://discourse.ubuntu.com/t/explanation/28829) </br> Concepts - discussion and clarification of key topics, architecture  | [Reference](https://discourse.ubuntu.com/t/reference/28828) </br> Technical information - specifications, APIs |
```

Comment:
By the way, why do we have two columns here?
On mobile, it looks a bit weird.
![Screen Shot 2023-06-23 at 12 11 15](https://github.com/anbox-cloud/discourse-docs/assets/98706738/c80de1af-8522-441a-9755-9abfb00bc65b)

Reply:
Alright, thanks for the explanation!